src/prop-types.js

```diff
@@ -245,10 +246,11 @@ export const OrgTemplateType = PropTypes.shape({
 // The “upcoming” status is used to indicate that the transaction is waiting
 // for another one to be mined before being processed.
 export const TransactionStatusType = PropTypes.oneOf([
-  TRANSACTION_STATUS_ERROR,
+  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_SIGNING,
   TRANSACTION_STATUS_PENDING,
   TRANSACTION_STATUS_SUCCESS,
-  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_ERROR,
```

Comment:
These array members were alphabetically sorted, is there a good reason to change that sort?

Reply:
See https://github.com/aragon/aragon/pull/1210/files/b795801d8d634b00462de36a82f9c4fa414b178c#r362507292